### PR TITLE
re-translation dreambooth.md

### PR DESCRIPTION
**Pull request by youyuanrsq**
The dreambooth.md has been retranslated and a bilingual version has been added

### CODE REVIEW

**Comment by xianbaoqian** [comment]
@zhongdongy @chenglu FYI

File: zh/dreambooth.md [hunk 1]
```diff
@@ -5,6 +5,7 @@ authors:
 - user: valhalla
 - user: pcuenq
 - user: 9of9
+- user: youyuanrsq
```
Comment (zhongdongy) [inline zh/dreambooth.md:8]: Please only put your username unser `authors` if you are one of the original writer, otherwise please only add to `translators`.

File: zh/dreambooth.md [hunk 2]
```diff
@@ -17,161 +18,211 @@ translators:
...
+[Dreambooth](https://dreambooth.github.io/) 是一种使用特殊的微调方式来教会 [Stable Diffusion](https://huggingface.co/blog/stable_diffusion) 新概念的技术。利用这个技术，有的人仅仅用他们自己很少的照片就将自己置身于奇妙的境界之中，而有些人则结合它生成新的风格。🧨Diffusers提供一个 [DreamBooth 训练脚本](https://github.com/huggingface/diffusers/tree/main/examples/DreamBooth)。使用这个脚本训练不会花费很长的时间，但是比较难筛选正确的超参数，并且容易过拟合。
 
-我们做了许多实验来分析不同设置下 DreamBooth 的效果。本文展示了我们的发现和一些小技巧来帮助你在用 DreamBooth 微调 Stable Diffusion 的时候提升结果。
+> [Dreambooth](https://dreambooth.github.io/) is a technique to teach new concepts to [Stable Diffusion](https://huggingface.co/blog/stable_diffusion) using a specialized form of fine-tuning. Some people have been using it with a few of their photos to place themselves in fantastic situations, while others are using it to incorporate new styles. [🧨 Diffusers](https://github.com/huggingface/diffusers) provides a Dreambooth [training script](https://github.com/huggingface/diffusers/tree/main/examples/dreambooth). It doesn't take long to train, but it's hard to select the right set of hyperparameters and it's easy to overfit.
```
Comment (zhongdongy) [inline zh/dreambooth.md:23]: Please don't include original English source in the translated file. You may add translator notes if necessary. Please kindly remove those contents.

File: zh/dreambooth.md [hunk 3]
```diff
@@ -17,161 +18,211 @@ translators:
...
+
+我们做了许多实验来分析不同参数配置下`DreamBooth`的效果。本文展示了我们的发现和一些小技巧来帮助你在用 `DreamBooth`微调`Stable Diffusion`的时候提升（生成图片的）效果。
```
Comment (zhongdongy) [inline zh/dreambooth.md:25]: Please make sure a single whitespace inserted between **inline code block** and **plain text** contents.

File: zh/dreambooth.md [hunk 4]
```diff
@@ -17,161 +18,211 @@ translators:
...
 
+> Before we start, please be aware that this method should never be used for malicious purposes, to generate harm in any way, or to impersonate people without their knowledge. Models trained with it are still bound by the [CreativeML Open RAIL-M license](https://huggingface.co/spaces/CompVis/stable-diffusion-license) that governs distribution of Stable Diffusion models.
+
 注意：该帖子的先前版本已出版为 [W＆B 报告](https://wandb.ai/psuraj/dreambooth/reports/Dreambooth-Training-Analysis--VmlldzoyNzk0NDc3)
 
-TL;DR: 推荐设置
------------
+> _Note: a previous version of this post was published [as a W&B report](https://wandb.ai/psuraj/dreambooth/reports/Dreambooth-Training-Analysis--VmlldzoyNzk0NDc3)_.
+
+## TL;DR: 推荐的设置（TL;DR: Recommended Settings）
```
Comment (zhongdongy) [inline zh/dreambooth.md:37]: Original contents should be stripped from titles as well.

**Review by zhongdongy — CHANGES_REQUESTED** [review]
Nice work and thanks! 🤗 But please make a few changes (mentioned in multiple comments) so we can merge your contributions.
